module/network/NetworkForwarder/data/config/NetworkForwarder.yaml

```diff
@@ -21,13 +22,13 @@ targets:
     message.input.Image: false # You probably don't want this unless you're testing something very weird
     message.output.CompressedImage: 10
 
-    message.vision.Balls: false
-    message.vision.Goals: false
-    message.vision.GreenHorizon: false
-    message.vision.Lines: false
-    message.vision.Obstacles: false
-    message.vision.VisualMesh: false
+    message.vision.Balls: true
+    message.vision.Goals: true
+    message.vision.GreenHorizon: true
+    message.vision.FieldLines: true
+    message.vision.Obstacles: true
+    message.vision.VisualMesh: true
```

Comment:
I'm not sure if I like this, I feel like it could go wrong easily. Like saturating the network. Does this work, does message.vision.Obstacles exist?